src/frontends/tensorflow_lite/src/sparsity_info.cpp

```diff
@@ -0,0 +1,163 @@
+        switch (m_data_desc[sparse_idx].segments_type) {
+        case ::tflite::SparseIndexVector_Uint8Vector:
+            read_sparse_data(m_data.data(),
+                             m_data.data() + total_size,
+                             row_size,
+                             static_cast<const ::tflite::Uint16Vector*>(m_data_desc[sparse_idx].indices)->values(),
+                             static_cast<const ::tflite::Uint8Vector*>(m_data_desc[sparse_idx].segments)->values());
+            break;
+        case ::tflite::SparseIndexVector_Uint16Vector:
+            read_sparse_data(m_data.data(),
+                             m_data.data() + total_size,
+                             row_size,
+                             static_cast<const ::tflite::Uint16Vector*>(m_data_desc[sparse_idx].indices)->values(),
+                             static_cast<const ::tflite::Uint16Vector*>(m_data_desc[sparse_idx].segments)->values());
+            break;
+        case ::tflite::SparseIndexVector_Int32Vector:
+            read_sparse_data(m_data.data(),
+                             m_data.data() + total_size,
+                             row_size,
+                             static_cast<const ::tflite::Uint16Vector*>(m_data_desc[sparse_idx].indices)->values(),
+                             static_cast<const ::tflite::Int32Vector*>(m_data_desc[sparse_idx].segments)->values());
+            break;
+        default:
+            FRONT_END_THROW("Unsupported vector type");
+            break;
+        }
+        break;
+    case ::tflite::SparseIndexVector_Int32Vector:
+        switch (m_data_desc[sparse_idx].segments_type) {
+        case ::tflite::SparseIndexVector_Uint8Vector:
+            read_sparse_data(m_data.data(),
+                             m_data.data() + total_size,
+                             row_size,
+                             static_cast<const ::tflite::Int32Vector*>(m_data_desc[sparse_idx].indices)->values(),
+                             static_cast<const ::tflite::Uint8Vector*>(m_data_desc[sparse_idx].segments)->values());
+            break;
+        case ::tflite::SparseIndexVector_Uint16Vector:
+            read_sparse_data(m_data.data(),
+                             m_data.data() + total_size,
+                             row_size,
+                             static_cast<const ::tflite::Int32Vector*>(m_data_desc[sparse_idx].indices)->values(),
+                             static_cast<const ::tflite::Uint16Vector*>(m_data_desc[sparse_idx].segments)->values());
+            break;
+        case ::tflite::SparseIndexVector_Int32Vector:
+            read_sparse_data(m_data.data(),
+                             m_data.data() + total_size,
+                             row_size,
+                             static_cast<const ::tflite::Int32Vector*>(m_data_desc[sparse_idx].indices)->values(),
+                             static_cast<const ::tflite::Int32Vector*>(m_data_desc[sparse_idx].segments)->values());
```

Comment:
Am I correct that we pack sparse tensor to ov.Tensor (that is dense)? What sizes are these dense tensors and will it be efficient by memory for a given model? May be we need to propagate this sparsity below and work on sparse tensors?